app/controllers/scenarios_controller.rb

```diff
@@ -6,6 +6,7 @@ def index
     set_table_sort sorts: %w[name public], default: { name: :asc }
 
     @scenarios = current_user.scenarios.reorder(table_sort).page(params[:page])
+    @glyph = session[:icon]
```

Comment:
I was hoping to save the icon temporarily and not have it be permanently stored in the db.